XmlSerializer in .NET 6 Omits Line Breaks by Default

mregen opened the pull request:
fixes #1759

opcfoundation-org commented:
The indent should be 2 spaces for my git diffs to work. 
If there are multiple places there should be a constant somewhere.

mregen commented:
> The indent should be 2 spaces for my git diffs to work. If there are multiple places there should be a constant somewhere.

added the same Utils functions as for the xml reader settings, added unit test, and hopefully have all places use the common settings.
